src/compiler/types.ts

```diff
@@ -1382,6 +1382,10 @@ module ts {
     /* @internal */ 
     export interface SymbolLinks {
         target?: Symbol;                    // Resolved (non-alias) target of an alias
+        aliasValueTarget?: Symbol;          // The result of resolving an alias as a value, which may be the target,

```

Comment:
i don't like comments like this when they're too long for  single line.  I'd just put the comment on top of the actual item itself.  (nit).